### PR TITLE
fix(Tabs): allow RTL scrolling

### LINKED ISSUE #9597: Tabs - RTL
The scroll arrow for [overflow tabs](https://patternfly-react-pr-9582.surge.sh/components/tabs#default-overflow-tabs) is not working in RTL mode.

Also (I'm guessing this is related?) tab overflow isn't being triggered when it should be:

LTR
image

RTL
image

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #9597

### CODE REVIEW

**Review by thatblindgeye — COMMENTED** [review]
This looks good. Only thing I'm noticing is that when I open the "default overflow" example in a new window, the scrolling doesn't work as expected when I enable RTL (looks like it's because the private `direction` is being reassigned only on mount). The example itself works fine on the main component page and RTL is enabled. Probably not a blocker, because other than our examples that offer an RTL switcher a user may not be swapping between an LTR and RTL language/direction on the fly, but wdyt?

**Review by thatblindgeye — COMMENTED** [review]
One other comment: the `leftScrollAriaLabel` and `rightScrollAriaLabel` props would need to be replaced with backward/forward props instead at some point.

**Comment by kmcfaul** [comment]
@thatblindgeye I added the direction assignment to the update method as well, which allows it to work when opened in a new window. The other case I'm thinking the update might come in handy is a language switcher element, which might not remount components on the page.

**Review by wise-king-sullyman — APPROVED** [review]
I'm still seeing the [scroll buttons show up when they shouldn't](https://patternfly-react-pr-9633.surge.sh/components/tabs#dynamic-tabs)

image


 but since that's a much smaller problem than the scrolling not working I won't block this PR over that.

**Comment by kmcfaul** [comment]
@wise-king-sullyman I'll dig into it a little, and will open a follow up to track it. The weird bit is that it doesn't seem consistent, checking the surge just now and it looks like its adding the scroll buttons at the proper time. 

@thatblindgeye I'll open the follow up, and will update the descriptions of the left/right label props for clarity in the meanwhile.

File: packages/react-core/src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -63,9 +63,9 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   isVertical?: boolean;
   /** Disables border bottom tab styling on tabs. Defaults to false. To remove the bottom border, set this prop to true. */
   hasNoBorderBottom?: boolean;
-  /** Aria-label for the left scroll button */
+  /** Aria-label for the back scroll button */
```
Comment (tlabaj) [inline packages/react-core/src/components/Tabs/Tabs.tsx:66]: can you deprecate this and the `rightScrollAriaLabel` prop and add `backScrollAriaLabel` and `forwardScrollAriaLabel`  props.

**Review by tlabaj — CHANGES_REQUESTED** [review]

File: packages/react-core/src/components/Tabs/Tabs.tsx (hunk 1 above)
Comment (kmcfaul) [reply packages/react-core/src/components/Tabs/Tabs.tsx]: Also updated some internal state to clean up the last bits of left/right

**Review by tlabaj — APPROVED** [review]
LGTM

**Pull request merged**